lib/assessment.js

```diff
@@ -32,6 +37,8 @@ var newArray2 = function newArray2(length, defaultsFunction) {
 
 };
 
+didn't even get this far, just need to see it again.  couldn't find antony's code along.  can you please update any repo with the code along so if we go over it again, it's not a blank with the starter code.  i will use my lunch time or after the day is over to go over this.  just let me know.
+
```

Comment:
Given my comment above, do you think you could come up with a solution to this question now?